aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -842,7 +842,7 @@ module aptos_framework::object {
     #[test(fx = @std)]
     fun test_correct_auid() {
         let auid1 = aptos_framework::transaction_context::generate_auid_address();
-        let bytes = aptos_framework::transaction_context::get_transaction_hash();
+        let bytes = aptos_framework::transaction_context::unique_session_hash();
```

Comment:
either you need a feature flag, or more simply - you can separate all usages of unique_session_hash() into separate PR that lands one release after that native function is introduced.

Reply:
@igor-aptos , since `get_transaction_hash` is still available, should `get_transaction_hash` still be used in this release instead of `unique_session_hash`?